src/languages/gams.js

```diff
@@ -78,7 +78,7 @@ export default function (hljs) {
       ASSIGNMENT,
       {
         className: 'comment',
-        begin: /([ ]*[a-z0-9&#*=?@>\\<:\-,()$\[\]_.{}!+%^]+)+/,
+        begin: /([ ]*[a-z0-9&#*=?@><:\-,()$[\]_.{}!+%^]+)+/,
```

Comment:
- The `[` does not need to be escape inside a `[` block... it's clear it's not opening a new block because it's inside one.
- `<` is not a character that's has ever needed escaping (so says my linter), and it's now invalid in `/u` mode.

> Like technically /{/ doesn't need to be escaped, but this PR is escaping it anyways.

Very different `{` **must be escaped** for the stricter UTF-8, which is the whole point of this PR.  `[` does not need to be.  The overall preference here is not to use unnecessary escapes anymore.

Reply:
> Very different `{` **must be escaped** for the stricter UTF-8, which is the whole point of this PR. `[` does not need to be. The overall preference here is not to use unnecessary escapes anymore. _They've always been flagged by the linter, it just hasn't been a high priority before now._

ah thanks for the explanation! Didn't realize that `{` had to be escaped for UTF-8 support. I just thought since it wasn't ambiguous it didn't need to be escaped.

---

> I'm a bit confused on this one. Is the removal of `\\` here correct? Is it trying to escape `<` or is `\` an actual character in this `[]` block?

Can confirm this? Being inside of a `[]` and regex, it looks to be like `\\` is for an `\` character and not trying to escape `<`. i.e. if this were a string originally, then I agree that it should be removed because it'd be `\<` but this has always been a regex so it doesn't look to me like it's escaping the `<`.

Reply:
Oh yikes, yeah I think you are right. :)  I put it back!